commands/comment.go

```diff
@@ -100,6 +112,10 @@ func commentOnReview(repo repository.Repo, args []string) error {
 		return errors.New("There is no matching review.")
 	}
 
+	if *commentParent != "" && !commentHashExists(*commentParent, r.Comments) {
```

Comment:
Is `*commentParent` necessarily a hash, or could it be some other reference resolvable to a hash?